File: python/resistor-color-duo/resistor_color_duo.py
"""Determine number for colors on resistance"""

COLORS = [
  'black',
  'brown',
  'red',
  'orange',
  'yellow',
  'green',
  'blue',
  'violet',
  'grey',
  'white',
]

def value(colors):
    return int(f'{COLORS.index(colors[0])}{COLORS.index(colors[1])}')

File: javascript/resistor-color-duo/resistor-color-duo.js
//
// This is only a SKELETON file for the 'Resistor Color Duo' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

const COLORS = [
  'black',
  'brown',
  'red',
  'orange',
  'yellow',
  'green',
  'blue',
  'violet',
  'grey',
  'white',
];

export const decodedValue = (colors) => {
  return Number(`${COLORS.indexOf(colors[0])}${COLORS.indexOf(colors[1])}`)
};

